### ISSUE TITLE
LDAP allowed user group doesn't work

### ISSUE THREAD

**Issue by roru69** [nilsteampassnet/TeamPass#2025]
### Steps to reproduce
1. Do a clean install of Teampass Master 5fd6c84b07c0af49cd4800b910b0ee629f91f7af
2. Add the AD parameters including the teampass group which should have access
3. Disable maintenance mode.
4. Log in as an AD user

### Expected behaviour
When the user is a member of the group, Teampass should create an account. When the user isn't a member of the group, Teampass shouldn't create an account.

### Actual behaviour
1. When I add the group in the setting"LDAP allowed user group" a line in tp.config.php at the bottom of the file is written, the line above that is missing a ",". So when you change tabs, the screen turns blank. because the config file is invalid.

    'ldap_new_user_is_administrated_by' => '0'
    'ldap_allowed_usergroup' => 'gg_teampass',

After solving this issue by adding the "," Teampass works again.

2. Log out the admin user. The "gg_teampass" Active Directory group is empty, no users are member of the group.
3. Log in as an Active Directory user.
4. Account is created. (This shouldn't happen. There are no members in "gg_teampass"

Question?
In the "LDAP allowed user group" setting should i use:
the "cn" which is: "gg_teampass" or should i use:
the "distinguised name" which is: "CN=gg_teampass,OU=GlobalGroups,OU=Groups,OU=Private,DC=mydomain,DC=local" ?

### Server configuration
**Operating system**:
Debian Jessie
**Web server:**
Apache
**Database:**
MariaDB
**PHP version:**
7.0.x
**Teampass version:** 
Teampass Master 5fd6c84b07c0af49cd4800b910b0ee629f91f7af
**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**
PLEASE attach to this message the file `/includes/config/tp.config.php`.

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
```
Insert your webserver log here
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by nilsteampassnet** [comment]
Can you please enable debug?

In `identify.php`, replace `$debugLdap = 0;` by `$debugLdap = 1;` and retry.

Share the debug file from folder `/files/`.

Thanks

**Comment by roru69** [comment]
Hi Nils,

I mailed you the 2 attachements.

1 is the config + debug file when i use the distinguised name attribue for the group gg_teampass
2 is the config + debug file when i use the CN attribute for the group gg_teampass


I don't see any line in the debug file for the "ldap_allowed_usergroup" setting. It looks like it isn't used in the script.

 Regards, Roru69

**Comment by nilsteampassnet** [comment]
Fix provided

**Comment by DFORRO** [comment]
Hi Nil. could you please provide fix to me also? We are facing to same issue with latest version.
Thanks

**Comment by nilsteampassnet** [comment]
Just published a new package for 2.1.27.10, please use it
